glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
>   ## Create
>  Creates the container: file system, namespaces, cgroups, capabilities.

When the namespaces have been created, it implies that at least one process has been started. (it is the `runc init` process for the runc implementation for example)

Although this process hasn't yet exec-ed to the container entrypoint, but the process did start.